src/cleanup_rules/swift/rules.toml

```diff
@@ -135,6 +136,54 @@ replace_node = "disjunction_expression"
 replace = "@lhs"
 is_seed_rule = false
 
+# Before 
```

Comment:
Can we note the intuition here that  `if_always_true` is assumed to run before `else_if_always_true`